.circleci/config.yml

```diff
@@ -413,7 +413,7 @@ executors:
     <<: *build-envs-oracle
 
   cucumber-with-mysql-ruby31: &cucumber-with-mysql-ruby31
-    resource_class: small
+    resource_class: medium
```

Comment:
This fails all the time, thanks to @akostadinov's efforts we know that it's because of OOM issues.

@akostadinov is investigating what is eating the memory, and whether it can be fixed (see https://github.com/3scale/porta/pull/4011), but in the meanwhile I want to bump the resource class for this executor.

It uses more run compute credits, but on the other hand, we spend so many credits due to constant reruns, that I think it might be compensated. In the end, if we improve the memory usage, we can drop it again.

I haven't touched the postgres executor, because we don't use it that often. oracle one is already on `large`.

Reply:
#4011 should be ready